pandas/core/groupby.py

```diff
@@ -2765,7 +2765,9 @@ def _get_grouper(obj, key=None, axis=0, level=None, sort=True,
     elif isinstance(key, BaseGrouper):
         return key, [], obj
 
-    if not isinstance(key, (tuple, list)):
+    # when MultiIndex, allow tuple to be a key
+    if not isinstance(key, (tuple, list)) or isinstance(key, tuple) and \
```

Comment:
use parens to avoid ambiguity when reading this condition

add an

```
is_axis_multiindex = isinstance(obj._info_axis, MultiIndex)
```

higher up and use it here (and elsewhere in this functionif you can)